lib/chef/provider/mount/linux.rb

```diff
@@ -29,10 +29,13 @@ class Linux < Chef::Provider::Mount::Mount
         # "findmnt" outputs the mount points with volume.
         # Convert the mount_point of the resource to a real path in case it
         # contains symlinks in its parents dirs.
+        def loop_mount_points
+          # get loop_mount_points only if not initialized earlier
+          @loop_mount_points ||= shell_out!("losetup -a").stdout
```

Comment:
We should probably `rescue Errno::ENOENT` and then return empty string here so that processing can continue without matching if losetup doesn't exist.

Reply:
did the change please check